Packs/NetscoutArborSightline/Integrations/NetscoutArborSightline/NetscoutArborSightline.py

```diff
@@ -0,0 +1,733 @@
+                          outputs_key_field='id',
+                          outputs=objects,
+                          readable_output=tableToMarkdown('Managed object list', hr,
+                                                          headers=MANAGED_OBJECTS_HR_HEADERS, removeNull=True),
+                          raw_response=raw_result)
+
+
+def tms_group_list_command(client: NetscoutClient, args: dict):
+    extend_data = argToBoolean(args.get('extend_data', False))
+    raw_result = client.tms_group_list()
+    data = raw_result.get('data')
+    data = data if isinstance(data, list) else [data]
+    hr = [build_human_readable(data=tms_group) for tms_group in data]
+    groups = [build_output(data=group, extend_data=extend_data) for group in data]
+    return CommandResults(outputs_prefix='NASightline.TMSGroup',
+                          outputs_key_field='id',
+                          outputs=groups,
+                          readable_output=tableToMarkdown('TMS group list', hr, removeNull=True),
+                          raw_response=raw_result)
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    try:
+        command = demisto.command()
+        params = demisto.params()
+
+        if not params.get('api_token') or not (api_token := params.get('api_token', {}).get('password')):
+            raise DemistoException('Missing API Key. Fill in a valid key in the integration configuration.')
+        base_url = urljoin(params['url'], 'api/sp')
+        verify_certificate = not params.get('insecure', False)
+        proxy = params.get('proxy', False)
+
+        if first_fetch_dt := arg_to_datetime(params.get('first_fetch', '3 days')):
+            first_fetch = first_fetch_dt.isoformat()
+        max_fetch = min(arg_to_number(params.get('max_fetch', 50)), 100)
+        alert_class = params.get('alert_class')
+        alert_type = params.get('alert_type')
+        classification = params.get('classification')
+        importance = IMPORTANCE_DICTIONARY.get(params.get('importance'))
+        importance_operator = params.get('importance_operator', '=')
+        ongoing = ONGOING_DICTIONARY.get(params.get('ongoing'))
+
+        demisto.debug(f'Command being called is {demisto.command()}')
+
+        headers: Dict = {
+            'X-Arbux-APIToken': api_token
+        }
+
+        client = NetscoutClient(
+            base_url=base_url,
+            verify=verify_certificate,
+            headers=headers,
+            proxy=proxy,
+            first_fetch=first_fetch,
+            max_fetch=max_fetch,
+            alert_class=alert_class,
+            alert_type=alert_type,
+            classification=classification,
+            importance=importance,
+            importance_operator=importance_operator,
+            ongoing=ongoing
+        )
+        args: dict = demisto.args()
+
+        result = ''
+        if command == 'test-module':
+            result = test_module(client)
+        elif command == 'fetch-incidents':
```

Comment:
with this setup:
![image](https://user-images.githubusercontent.com/31018228/115995072-70e24100-a5e2-11eb-86cd-d19b7525e529.png)
getting:
```
Error: [Traceback (most recent call last):
   
   File "<string>", line 733, in <module>
   File "<string>", line 728, in main
   File "<string>", line 5358, in return_error
 Exception: Failed to execute fetch-incidents command.
Error:
Error in API call [422] - UNPROCESSABLE ENTITY: 
tms is not a valid alert_class
```

Reply:
Interesting looks like a bug in their API (also happens for class `system_error`) 
![image](https://user-images.githubusercontent.com/44666568/116152118-3239ad00-a6ee-11eb-991a-215a70154e67.png)
Contacting the vendor.